apps/web/playwright/onboarding.test.ts

```diff
@@ -11,17 +11,4 @@ test.describe("Onboarding", () => {
       },
     });
   });
-
```

Comment:
There was a test validating if param on query was placed on the input above, but removing since it's no longer needed